learn/how-tos/create-prefab-with-partials.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "How to Create Partials in a Prefab"
+id: ""
+sidebar_label: "Creating Partials in a Prefab"
+---
+---
+In this document, the following two use cases explain how to use [Partials in a Prefab](/learn/app-development/custom-widgets/prefab-with-partials).
+
+1. [Partials in Web and Mobile](#partials-in-web-and-mobile)
+2. [Using Partial for Data Table with Row Expansion](#data-table-with-row-expansion)
+
+## Partials in Web and Mobile
+
+To build a Prefab for the web and mobile, you develop different partials for different viewports, and it segregates the code automatically.
```

Comment:
Doesn't "segregates the code automatically" sound like it automatically handles the show and hide conditions for different viewports ? "You develop different partials for different viewports, which helps in segregating the code." ?

Reply:
Well pointed!